airflow/dags/sync_ntd_data_xlsx/ridership/historical_raw_monthly_ridership_no_adjustments_or_estimates.yml

```diff
@@ -0,0 +1,5 @@
+operator: operators.NtdDataProductXLSXOperator
+
+product: 'raw_monthly_ridership_no_adjustments_or_estimates'
+xlsx_file_url: 'https://www.transit.dot.gov/sites/fta.dot.gov/files/2024-09/S%26S%20Time%20Series-May%202024-Major%20Only_240903.xlsx'
```

Comment:
I believe this file changes monthly. Are we planning on accounting for this and creating the needed automation for monthly downloads or is that something for a future effort?

Reply:
Hey Evan! Yep – we saw that all of these datasets are being updated regularly, even annual reports for previous years, so automation is built into this work. I just pushed changes with a placeholder value for the schedule - the first day of the month, every month. We can update that from here, but figured that was a reasonable suggestion. Let me know if you have any other thoughts.

Reply:
Hey @evansiroky – this PR is ready for merge, but my last remaining question relates to the scheduling of the DAG tasks in this PR. Based on the frequency that we see a lot of the endpoints updating (~monthly) I configured the DAG tasks to run once a month (first day of the month, every month, 3am PT), but open to suggestion here.